src/transaction/atomic_replication_helper.cpp

```diff
@@ -499,6 +857,45 @@ namespace cublog
       }
   }
 
+  bool
+  atomic_replication_helper::atomic_log_sequence::atomic_log_entry::is_control () const
+  {
+    return (m_rectype == LOG_START_ATOMIC_REPL ||
+	    m_rectype == LOG_END_ATOMIC_REPL ||
+	    m_rectype == LOG_SYSOP_ATOMIC_START ||
+	    m_rectype == LOG_SYSOP_END ||
+	    m_rectype == LOG_SYSOP_START_POSTPONE);
```

Comment:
Could you tell me why `LOG_SYSOP_START_POSTPONE` has to be regarded as a control log? There must be a reason, but I feel like it just complicates the design.

Reply:
See section (3) of the comment for the `class atomic_replication_helper` - with the layout encountered during a vacuum transaction.

Reply:
To be re-investigated during - http://jira.cubrid.org/browse/LETS-541 - see comment - http://jira.cubrid.org/browse/LETS-541?focusedCommentId=4762726&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4762726 .